core/src/main/java/org/apache/spark/io/NioBufferedFileInputStream.java

```diff
@@ -32,8 +34,11 @@
  */
 public final class NioBufferedFileInputStream extends InputStream {
 
+  private static final Cleaner CLEANER = Cleaner.create();
```

Comment:
[a9f3025](https://github.com/apache/spark/pull/43333/commits/a9f3025afb2381f573ee0c3e146da9d0b704593b) rename this from `cleaner` to `CLEANER` due to it's static